diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index be9df86be09c7655787a65c55ae6da01858894c3..18cd0141989c516b49b134507072e23f47317e2a 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -260,4 +260,7 @@ function expand_(str, max, maxLength, isTop) {
     }
     return acc;
 }
+const commonJsExport = Object.assign(expand, exports, { default: expand });
+Object.defineProperty(commonJsExport, "__esModule", { value: true });
+module.exports = commonJsExport;
 //# sourceMappingURL=index.js.map
